.github/workflows/ci_linux_x64_nogil-libshortfin.yml

```diff
@@ -79,6 +79,7 @@ jobs:
         mkdir ${{ env.LIBSHORTFIN_DIR }}/build
         cd ${{ env.LIBSHORTFIN_DIR }}/build
         cmake -GNinja \
+          -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Any particular reason for `Debug` here and only here? Or can this be reverted?

Reply:
Yes - we test on the CI with Debug/Asserts. This job was added as a release build... not sure why.